### DISCUSSION TITLE
Is there still going to be sync support?

### DISCUSSION THREAD

**Discussion by Fragaile** [kaliiiiiiiiii/Selenium-Driverless#74] [Q&A] [answered]
By sync I mean using this library without asyncio. Is that still supported? I can't get it to work on the dev installation but maybe it will still be supported on the main version? Thanks!

**Comment by ttlns** [comment]
@Fragaile please provide the script you tried with. And yes, it should still be supported.

**Reply by ttlns** [reply]
and the error//unexpected behaviour as well of course: )

**Reply by Fragaile** [reply]
```
from selenium_driverless import webdriver


options = webdriver.ChromeOptions()

with webdriver.Chrome(options=options) as driver:
    driver.get('https://google.com/')
```

```c:\Users\User\Desktop\Programming\Python\Test\test4.py:9: RuntimeWarning: coroutine 'Chrome.get' was never awaited
  driver.get('https://google.com/')
RuntimeWarning: Enable tracemalloc to get the object allocation traceback
Press any key to continue . . . 
```

And it doesn't launch, just tells me to "enable tracemalloc to get the object allocation traceback"

**Reply by ttlns** [reply, marked as answer]
use `from selenium_driverless.sync import webdriver` haha:)

**Reply by Fragaile** [reply]
Dude.. I'm such an idiot.. thank you once again 😂💖

**Reply by ttlns** [reply]
sure, now worries haha:)
